snuba/consumers/consumer.py

```diff
@@ -355,6 +360,66 @@ def build_writer() -> ProcessedMessageBatchWriter:
     return build_writer
 
 
+class DeadLetterStep(
+    ProcessingStep[Tuple[StorageKey, Union[None, BytesInsertBatch, ReplacementBatch]]]
+):
+    def __init__(self, producer: AbstractProducer, topic: Topic):
+        self.__producer = producer
+        self.__topic = topic
+        self.__futures: Deque[Future] = deque()
+        self.__closed = False
+
+    def __format_payload(
+        self,
+        message: Message[
+            Tuple[StorageKey, Union[None, BytesInsertBatch, ReplacementBatch]]
+        ],
+    ) -> Optional[KafkaPayload]:
+        kafka_payload = None
+        storage_key, payload = message.payload
+        if isinstance(payload, BytesInsertBatch):
```

Comment:
We won't be handling `ReplacementBatch`'s right? since those go to the replacements topic yeah?

Reply:
yep ignore replacements